packages/app-extrinsics/src/index.tsx

```diff
@@ -29,3 +50,7 @@ export default class ExtrinsicsApp extends React.PureComponent<Props> {
     );
   }
 }
+
+export { ExtrinsicsApp };
+
+export default withObservableBase(accountsObservable.subject, { propName: 'accountAll' })(ExtrinsicsApp);
```

Comment:
The issue with the `t` return is here - if you wrap this in a `translate()` it will inject the `t` property. As an example, see https://github.com/polkadot-js/apps/blob/master/packages/app-accounts/src/Editor.tsx#L232